<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleFileResourceConfigWidget</class>
 <widget class="QWidget" name="SingleFileResourceConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>350</height>
   </size>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>File</string>
      </attribute>
      <layout class="QVBoxLayout" name="tabLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="filenameLabel">
           <property name="text">
            <string>File Name:</string>
           </property>
           <property name="buddy">
            <cstring>kcfg_Path</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="KUrlRequester" name="kcfg_Path"/>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="pathLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Select a file. A new file will be created if it doesn't exist. You can also use a URL, but in that case, monitoring for file changes is not possible.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="buddy">
            <cstring>kcfg_Path</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="displayNameLabel">
           <property name="text">
            <string>Display Name:</string>
           </property>
           <property name="buddy">
            <cstring>kcfg_DisplayName</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="kcfg_DisplayName"/>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="kcfg_ReadOnly">
           <property name="text">
            <string>Read only</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="readOnlyLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="visible">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>When read-only mode is enabled, no changes will be written. Read-only mode is automatically enabled when the selected file does not support write access.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="buddy">
            <cstring>kcfg_ReadOnly</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QCheckBox" name="kcfg_MonitorFile">
           <property name="text">
            <string>Enable file &amp;monitoring</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="monitoringLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Changes made by other applications will reload the file when monitoring is enabled. Backups will be created to avoid conflicts when possible.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="buddy">
            <cstring>kcfg_MonitorFile</cstring>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="KMessageWidget" name="statusLabel"/>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>138</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KMessageWidget</class>
   <extends>QFrame</extends>
   <header>kmessagewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
